src/main/java/htsjdk/variant/vcf/VCFHeader.java

```diff
@@ -270,7 +270,9 @@ public SAMSequenceDictionary getSequenceDictionary() {
 
         final List<SAMSequenceRecord> sequenceRecords = new ArrayList<SAMSequenceRecord>(contigHeaderLines.size());
         for (final VCFContigHeaderLine contigHeaderLine : contigHeaderLines) {
-            sequenceRecords.add(contigHeaderLine.getSAMSequenceRecord());
+            final SAMSequenceRecord samSequenceRecord = contigHeaderLine.getSAMSequenceRecord();
```

Comment:
How do you end up with nulls here?  Shouldn't it be returning lines with UNKNOWN_SEQUENCE_LENGTH?